schedule/kernel/agama_install_ltp_baremetal.yaml

```diff
@@ -0,0 +1,12 @@
+name: agama_install_ltp_pvm
+description:    >
+    Install system and LTP using the Agama installer on PowerVM.
+vars:
+    INST_AUTO: kernel/agama_auto/sle16.jsonnet
+schedule:
+    - installation/ipxe_install
```

Comment:
Profile bellow has `installation/bootloader_start`

Reply:
We should probably have unified profile for power and ipmi, like we have for SLE15 with backend condition.

Reply:
rather use `- installation/bootloader_start` as in  `schedule/kernel/agama_prepare_baremetal.yaml`